src/store/actions/index.js

```diff
@@ -178,6 +182,43 @@ export const getTenants = () => {
   };
 };
 // ------------------------------------------------|
+// GET TENANT -----------------------------------|
+// Takes in the tenant id to pass in as a url
+// parameter for endpoint
+export const getTenant = id => async dispatch => {
+  dispatch({ type: GET_TENANT_ID_START });
+
+  try {
+    const res = await axiosAuth().get(`${baseUrl}/tenants/${id}`);
+
+    dispatch({
+      type: GET_TENANT_ID_SUCCESS,
+      payload: {
+        property: res.data
```

Comment:
Property is ghosting in the payload object. Might be best to set `payload: res.data` here so that when accessing the data in the reducer it would be `action.payload` instead of `action.payload.tenant`